integrations/gradient/src/haystack_integrations/components/embedders/gradient/gradient_document_embedder.py

```diff
@@ -125,6 +135,7 @@ def _generate_embeddings(self, documents: List[Document], batch_size: int) -> Li
     def run(self, documents: List[Document]):
         """
         Embed a list of Documents.
+
         The embedding of each Document is stored in the `embedding` field of the Document.
 
         :param documents: A list of Documents to embed.
```

Comment:
missing that returns with documents for example see this [one](https://github.com/deepset-ai/haystack/blob/main/haystack/components/connectors/openapi_service.py#L91)

Reply:
👍🏽